Makefile

```diff
@@ -25,6 +25,7 @@ rhel:
 	gpg --dearmor < keys/verifier-public-key-redhat-release > "$$keydir/verifier-public-key-redhat.gpg"; \
 	gpg --dearmor < keys/verifier-public-key-redhat-beta-2 >> "$$keydir/verifier-public-key-redhat.gpg"; \
 	gpg --enarmor < "$$keydir/verifier-public-key-redhat.gpg" > "$$keydir/verifier-public-key-redhat"; \
+	sed -i sed -i 's/ARMORED FILE/PUBLIC KEY BLOCK/' "$$keydir/verifier-public-key-redhat"; \
```

Comment:
`sed -i sed -i` ??? seems like a typo

Reply:
Oops, push a fix to my branch or replace with a parallel PR?

Reply:
ok, fixed with 55054e6 -> eab0d87.